CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# Version 1.4.6 
+- Fixed an issue affecting prechat forms, that would stop a visitor from starting a chat.
```

Comment:
- Fixed an issue affecting pre-chat forms, that would stop a visitor from starting a chat.